<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>mru</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SampleScope</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/icons/icons/utilities-system-monitor.svg</normaloff>:/icons/icons/utilities-system-monitor.svg</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="ModeControl" name="modeControl" native="true">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="GraphControl" name="graphControl" native="true">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QFrame" name="frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="_2">
           <property name="margin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QSplitter" name="splitter">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="childrenCollapsible">
              <bool>false</bool>
             </property>
             <widget class="GraphNormal" name="graphNorm" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>100</height>
               </size>
              </property>
             </widget>
             <widget class="GraphFFT" name="graphFFT" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>100</height>
               </size>
              </property>
             </widget>
             <widget class="GraphXY" name="graphXY" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>100</height>
               </size>
              </property>
             </widget>
             <widget class="GraphAutoCorr" name="graphAutoCorr" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>100</height>
               </size>
              </property>
             </widget>
             <widget class="MeasurementWidget" name="widgetMeasurements" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>100</height>
               </size>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollBar" name="dialTriggerLevel">
        <property name="maximum">
         <number>4095</number>
        </property>
        <property name="value">
         <number>2000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="invertedAppearance">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutChannelControl">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="ChannelControl" name="channelControlCh1" native="true">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ChannelControl" name="channelControlCh2" native="true">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="TriggerControl" name="triggerControl" native="true"/>
        </item>
        <item>
         <widget class="TimeControl" name="timeControl" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1066</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuHardware">
     <property name="title">
      <string>Hardware</string>
     </property>
     <addaction name="actionUse_real_hardware"/>
     <addaction name="actionUse_software_emulation"/>
     <addaction name="actionAuto_connect"/>
     <addaction name="actionConnect"/>
     <addaction name="actionDisconnect"/>
    </widget>
    <addaction name="menuHardware"/>
    <addaction name="actionCalibration_Dialog"/>
    <addaction name="actionExport_Data"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAuto_connect"/>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="separator"/>
   <addaction name="actionUse_real_hardware"/>
   <addaction name="actionUse_software_emulation"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionExport_Data">
   <property name="text">
    <string>Export Data</string>
   </property>
  </action>
  <action name="actionCalibration_Dialog">
   <property name="text">
    <string>Calibration Dialog</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionUse_real_hardware">
   <property name="text">
    <string>Use real hardware</string>
   </property>
  </action>
  <action name="actionUse_software_emulation">
   <property name="text">
    <string>Use software emulation</string>
   </property>
  </action>
  <action name="actionAuto_connect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto connect</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ModeControl</class>
   <extends>QWidget</extends>
   <header>modecontrol.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChannelControl</class>
   <extends>QWidget</extends>
   <header>channelcontrol.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TriggerControl</class>
   <extends>QWidget</extends>
   <header>triggercontrol.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TimeControl</class>
   <extends>QWidget</extends>
   <header>timecontrol.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GraphFFT</class>
   <extends>QWidget</extends>
   <header>graphfft.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GraphNormal</class>
   <extends>QWidget</extends>
   <header>graphnormal.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GraphXY</class>
   <extends>QWidget</extends>
   <header>graphxy.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GraphControl</class>
   <extends>QWidget</extends>
   <header>graphcontrol.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MeasurementWidget</class>
   <extends>QWidget</extends>
   <header>measurementwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GraphAutoCorr</class>
   <extends>QWidget</extends>
   <header>graphautocorr.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
